src/pybind/mgr/dashboard_v2/tests/test_auth.py

```diff
@@ -25,7 +25,7 @@ def setup_server():
         cherrypy.tree.mount(auth, "/api/auth")
         cherrypy.tree.mount(Ping(), "/api/test",
                             config={'/': {'tools.autenticate.on': True}})
-        module.set_localized_config('session-expire','2')
+        module.set_localized_config('session-expire','0.5')
```

Comment:
why change this value? `0.5` seems very small and might cause false negatives when running this unit test

Reply:
Because [unit tests should be fast](https://github.com/ghsukumar/SFDC_Best_Practices/wiki/F.I.R.S.T-Principles-of-Unit-Testing#fast).